content/source/docs/cloud/workspaces/variables.html.md

```diff
@@ -1,130 +1,106 @@
+Terraform Cloud workspace variables let you customize configurations, modify Terraform's behavior, and store information like provider credentials. You can set variables specifically for each workspace or you can create variable sets to reuse the same variables across multiple workspaces. Terraform Cloud applies workspace variables to all runs within that workspace.
 
+You must have [`read variables` permission](/docs/cloud/users-teams-organizations/permissions.html#general-workspace-permissions) to view the variables for a particular workspace and to view the variable sets in your organization. Once you have the proper [read and write variables permissions](/docs/cloud/users-teams-organizations/permissions.html#general-workspace-permissions), you can create and edit both workspace-specific variables and variable sets through:
 
+- The [Terraform Cloud UI](/docs/cloud/workspaces/managing-variables.html).
+- The Variables API for [workspace-specific variables](/docs/cloud/api/workspace-variables.html) and [variable sets](/docs/cloud/api/variable-sets.html).
+- The `tfe` provider [`tfe_variable`](https://registry.terraform.io/providers/hashicorp/tfe/latest/docs/resources/variable) resource, which can be more convenient for large numbers of complex variables.
 
-## Loading Variables from Files
-
-If a workspace is configured to use Terraform 0.10.0 or later, you can commit any number of [`*.auto.tfvars` files](/docs/language/values/variables.html#variable-files) to provide default variable values. Terraform will automatically load variables from those files.
-
-If any automatically loaded variables have the same names as variables specified in the Terraform Cloud workspace, the workspace's values will override the automatic values.
-
-You can also use the optional [Terraform Cloud Provider](https://registry.terraform.io/providers/hashicorp/tfe/latest/docs/resources/variable) to update a workspace's variables. This has the same effect as managing workspace variables manually or via the API, but can be more convenient for large numbers of complex variables.
+[permissions-citation]: #intentionally-unused---keep-for-maintainers
 
-## Variable Limits
 
-The following limits apply to variables:
 
-Component   |  Limit
-------------|---------------------------
-description |  512 characters
-key         |  128 characters
-value       |  256 kilobytes
+## Types
 
-## Managing Variables in the UI
+You can create both Terraform variables and environment variables in Terraform Cloud.
 
-To view and manage a workspace's variables, navigate to that workspace and click the "Variables" navigation link at the top.
+> **Hands-on:** The [Create Infrastructure tutorial](https://learn.hashicorp.com/tutorials/terraform/cloud-workspace-configure?in=terraform/cloud-get-started) on HashiCorp Learn shows how to set both Terraform variables and provider credential environment variables.
```

Comment:
@ritsok modified this tutorial flow a bit, so I don't think this is accurate. Can we update this?

https://learn-git-addvarsettfcgs-hashicorp.vercel.app/tutorials/terraform/cloud-create-variable-set

Reply:
yes! thank you - updated!